lib/toolshed/top_server.ex

```diff
@@ -0,0 +1,252 @@
+defmodule Toolshed.TopServer do
+  @moduledoc """
+  Find the top processes and refresh them after every 1000 milliseconds
+  """
```

Comment:
```suggestion
  @moduledoc false
```
This module isn't meant to be used by Toolshed users, so turn off the module docs so they don't see it in.